jekyll-assets/_layouts/boxes404.html

```diff
@@ -0,0 +1,40 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    {% include head.html %}
+    <link rel="stylesheet" href="{{ site.baseurl }}/css/boxes.css">
+  </head>
+  <body>
+    {% include fonts.html %}
+    {% include header.html %}
+    {% include toptitle404.html %}
+    {% include subtitle404.html %}
```

Comment:
Having a separate layout makes sense to me, perhaps it's not worth extracting `toptitle404.html` or `subtitle404.html` though and instead inline those here, e.g.

```suggestion
    <div class="toptitle">
      <h1><strong>404</strong></h1>
      <div id="search-container">
        <input type="text" id="search-input" placeholder="search...">
        <ul id="results-container"></ul>
      </div>
    </div>
    <div class="subtitle">
      <p>We couldn't find what you were<br /> looking for, please browse our updated<br /> <strong>Raspberry Pi</strong> Documentation</p>
    </div>
```

This could then be `jekyll-assets/_layouts/404.html`?

Reply:
Comma splice. You need to replace the comma with a full stop here, and capitalise the following `P` on `please`.

Also, I'm not wild about the phraseology here. Do we actually need anything other than `Page not found.` then a gap, then the usual `Raspberry Pi Documentation` heading?

Reply:
> Also, I'm not wild about the phraseology here. Do we actually need anything other than Page not found. then a gap, then the usual Raspberry Pi Documentation heading?

That's @jackbenwillis' decision to make.

Reply:
Already fixed https://github.com/raspberrypi/documentation/pull/2026/commits/1279d9a744ff486d29621108403babaf9695bc61